### PR TITLE
add githunt to project list

### PR DESCRIPTION
**Pull request by SriNandan33**

### CODE REVIEW

File: _data/projects/githunt.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- python
+- click
+- colorama
+- cli
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/SriNandan33/githunt/labels/up%20for%20grabs
+stats:
+  issue-count: 1
+  last-updated: '2020-01-18T13:53:58.907Z'
```
Comment (ritwik12) [inline _data/projects/githunt.yml]: @SriNandan33 You don't need to include this. it will be automatically added later.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/githunt.yml (hunk 1 above)
Comment (SriNandan33) [reply _data/projects/githunt.yml]: Hoo okay. Should I remove the key also `last-updated`? or just the timestamp value and keep the empty string (last_updated: '' )?

Comment (ritwik12) [reply _data/projects/githunt.yml]: remove all three lines, 13, 14, 15 @SriNandan33

Comment (SriNandan33) [reply _data/projects/githunt.yml]: Okay, Thanks @ritwik12

Comment (SriNandan33) [reply _data/projects/githunt.yml]: Removed lines 13,14,15 @ritwik12 .

**Pull request merged**